sss/geoid.py

```diff
@@ -0,0 +1,117 @@
+import os
+import glob
+
+import pandas as pd
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Float,
+    Boolean,
+    ForeignKey)
+
+from .base import Base, AutomappedDB, DeclarativeDB
+from .base import db_url as default_db_url
+#from sss import GEOID
+
+
+# declare GEOID data columns and data type
+class GEOID(Base):
+    '''
+    state: str
+        name of the state, e.g. WA
+    place: str
+        SSS place name
+    state_fips: str
+        fips code of state
+    county_fips: str
+        fips code of county
+    place_fips: str
+        fips code of place
+    cpi_region: str
+        name of cip region
+
+    '''
+    __tablename__ = 'geo_identifier'
+    state = Column('state', String ,primary_key = True)
+    place = Column('place', String , primary_key = True)
+    #state = Column('state', String, ForeignKey("self_sufficiency_standard.state"),primary_key = True)
+    #place = Column('place', String, ForeignKey("self_sufficiency_standard.place"), primary_key = True)
+    state_fips = Column('state_fips',String)
+    county_fips = Column('county_fips', String)
+    place_fips =  Column('place_fips', String)
+    cpi_region = Column('cpi_region', String)
+    
+def geo_identifier_creator(county_table, cpi_table):
+    """The function to create geoidentifier
+   
+   Parameters
+    ----------
+    county_table : string
+            The path of the county table including FIPS code
+    cpi_table : string
```

Comment:
I'd be inclined to rename these variables to `county_file` and `cpi_file` since they are files that you are passing in (not SQL tables or data frames).